src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1004,6 +1004,28 @@ static bool processAreaScopeV2(const Scope* scoP, BSONObj &areaQuery)
 #define OPR_NOT_EXIST "NOT EXIST"
```

Comment:
The following line in CNR should be modified:

Add: dates support in attribute values (Issue #1038)

to 

Add: dates support in attribute values and q filters (Issue #1039)

(Note also the fix in the issue name, currently it is working using #1038).